src/autocomplete/index.jsx

```diff
@@ -67,7 +75,12 @@ export default function AutoComplete(props) {
           inputValue: renderLabel,
           suggestion: renderLabel
         }}
-        onConfirm={option => setValue(option ? option.value : '')}
+        onConfirm={option => {
+          if (typeof option === 'string') {
+            return setValue(option);
+          }
+          setValue(option.value || '');
+        }}
```

Comment:
Would prefer this to be extracted now it's multi-line but not gonna die on that hill.

Reply:
cheers for pointing out I'd actually changed how that works 😬

Reply:
reverted back to `option ?...`